app/src/org/commcare/network/HttpUtils.java

```diff
@@ -18,29 +18,23 @@
     public static String getCredential(@Nullable Pair<String, String> usernameAndPasswordToAuthWith) {
         String credential;
         if (usernameAndPasswordToAuthWith == null) {
```

Comment:
Earlier when `usernameAndPasswordToAuthWith` was passed as `null` we weren't using `buildAppPassword` to obfuscate the password.